CMSgov/easi-app
# [EASI-2415] TRB admin home wrapper

# EASI-2415

## Changes and Description

[Figma
](https://www.figma.com/file/iGatgTnlzTuqW9CQFYHDam/TRB-Intake-e2e?node-id=193%3A15620)

- File structure and routes (see below) for TRB admin home
- TRB admin home wrapper ([src/views/TechnicalAssistance/AdminHome/index.tsx](https://github.com/CMSgov/easi-app/blob/7549ad3876cbd9d46bc32b058f310b74a35626a4/src/views/TechnicalAssistance/AdminHome/index.tsx))
  - Side navigation
  - Stub components for each sub page
- Summary component ([src/views/TechnicalAssistance/AdminHome/Summary/index.tsx](https://github.com/CMSgov/easi-app/blob/7549ad3876cbd9d46bc32b058f310b74a35626a4/src/views/TechnicalAssistance/AdminHome/Summary/index.tsx))
- Testing

#### Routes
- Request home - `/trb/:id/request`
- Initial request form - `/trb/:id/initial-request-form`
- Supporting documents - `/trb/:id/documents`
- Feedback - `/trb/:id/feedback`
- Advice letter - `/trb/:id/advice`
- Notes - `trb/:id/notes`



## How to test this change

- Log in as user with `EASI_TRB_ADMIN_D` job code
- Create TRB request
- Go to `/trb/:id/request` to view as admin

**Note:** if you are viewing a draft request that was created by a different user AND saved before the attendees section of the form was completed, only their EUA will show up in the Requester portion of the summary component. Once the form has been completed, it should show the requester name and component acronym.

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
